src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -2684,8 +2684,8 @@ void SimAirLoop(EnergyPlusData &state,
 
     // SUBROUTINE LOCAL VARIABLE DEFINITIONS
 
-    auto &PrimaryAirSystems(state.dataAirSystemsData->PrimaryAirSystems);
-    auto &AirLoopControlInfo(state.dataAirLoop->AirLoopControlInfo);
+    auto &PrimaryAirSystems = state.dataAirSystemsData->PrimaryAirSystems;
+    auto &AirLoopControlInfo = state.dataAirLoop->AirLoopControlInfo;
```

Comment:
Note the first CppCheck suggestion is valid, the second is not.

[src/EnergyPlus/SimAirServingZones.cc:2687]:(style),[constVariable],Variable 'PrimaryAirSystems' can be declared as reference to const
[src/EnergyPlus/SimAirServingZones.cc:2688]:(style),[constVariable],Variable 'AirLoopControlInfo' can be declared as reference to const

Reply:
Interesting. Again, I didn't do this type of analysis unless the scope was very tiny and I could tell at a glance that there were no modifications. Either way, these shortcuts are temporary and will be replaced by `auto &thisPrimaryAirSystem` in a near-future pass either by me or someone else (probably you!)